AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiCalls/AuthService.swift

```diff
@@ -38,5 +38,23 @@ class AuthService {
                 }
         }
     }
-     
+    
+    // MARK: This function will be removed after reviewing Pull Request
+    
+    func oauthExample(username: String! , password: String!, completion: @escaping (_ accessToken: String?) -> Void) {
+        Network.shared.post(ApiEndPoints.authenticate(), params: ApiConfig.oauthCredentials(username: username, password: password), withHeader: true, completion: { parsedJSON, statusCode, error in
+            if error != nil {
+                debugPrint(error.debugDescription)
+                completion(nil)
+                return
+            }
+            guard let data = parsedJSON else {
+                completion(nil)
+                return
+            }
+            let userAccessToken = data["access_token"].string
```

Comment:
I won't advice plucking data from JSON like this. We can create a model class for the response and EVReflection handles deserialization so it is easy to manage if we need to make changes.